Makefile

```diff
@@ -165,9 +165,12 @@ generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
 
 .PHONY: release-manifests
 release-manifests: ## Builds the manifests to publish with a release
+    ifndef VERSION
```

Comment:
indent here is still off. I think the `ifndef VESION` can be same indent as `release-manifest`, but the error line has to match indent with the rest of the target